ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2022.3.14f1
-m_EditorVersionWithRevision: 2022.3.14f1 (eff2de9070d8)
+m_EditorVersion: 2022.3.22f1
```

Comment:
latest 2022 LTS is .25: is there any reason why we stopped 3 versions earlier?

Reply:
The [last PR](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/884) for version bumps took longer than usual to review, and it's only just been a week since that was merged.
So, I figure let's get this release out the door to avoid version bumping adding more time for a release.
What's nice at least is that all the [tests](https://unity-ci.cds.internal.unity3d.com/job/36362032/dependency-graph/overview?r=36362032) ran on the latest LTS (even 2022.3.26f1).